src/lib.rs

```diff
@@ -77,6 +80,64 @@ pub enum Error {
 
 type Result<T> = std::result::Result<T, Error>;
 
+#[no_mangle]
+pub extern "C" fn Sanity() -> i32 {
```

Comment:
I don't think C bindings should be part of this crate, or at least not in lib.rs.

Reply:
but AirAPI_Windows has it? And it works with Unity game engine (which is why the PR is from an HPVDT repo)

Can we set up a multi-module project and only publish the non-extern part?

Reply:
I'm not against a C binding, but it should be in a separate module, behind a `feature` that's off by default (because most people using this crate will not need it).

Or you could create your own repository, a simple cdylib crate which imports this crate and adds the bindings (which should be relatively simple almost one-liners). You could have your other customizations, like the rust-toolchain.toml file, and the cargo.lock there too, and this MR would just be about adding the fusion part.